letsencrypt/plugins/manual.py

```diff
@@ -90,6 +90,8 @@ def __init__(self, *args, **kwargs):
     def add_parser_arguments(cls, add):
         add("test-mode", action="store_true",
             help="Test mode. Executes the manual command in subprocess.")
+        add("public-ip-logging-ok", action="store_true",
```

Comment:
Small suggestion: This becomes the flag `--manual-public-ip-logging-ok`. This is very verbose. What about `--manual-ip-logging-ok` or `--manual-public-log-ok`?